### PR TITLE
[DM-3888] [DM-3915] [DM-3989] [GRAFT] [y2024] - lwm2m changelogs

### PR DESCRIPTION
**Pull request by jakubsobolewskisag**
graft of a change logs to y2024

### CODE REVIEW

File: content/change-logs/device-management/cumulocity-10-18-540-210-improvement-lwm2m-empty-payload-no-ct.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+date:
+title: LWM2M agent can now handle an empty payload without content-type data
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/cumulocity-10-18-540-210-improvement-lwm2m-empty-payload-no-ct.md:3]:
```suggestion
title: Allowing LWM2M device to send empty payload without content-type data
```

File: content/change-logs/device-management/cumulocity-10-18-540-210-improvement-lwm2m-empty-payload-no-ct.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    label: Improvement
+product_area: Device Management & Connectivity
+component:
+  - value: component-1KLUzmqfe
+    label: LWM2M
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: DM-3989
+version: 10.18.540.210
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/cumulocity-10-18-540-210-improvement-lwm2m-empty-payload-no-ct.md:15]:
```suggestion
version: 10.18.540.209
```

File: content/change-logs/device-management/cumulocity-10-18-540-210-improvement-lwm2m-location-update.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    label: Improvement
+product_area: Device Management & Connectivity
+component:
+  - value: component-1KLUzmqfe
+    label: LWM2M
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: DM-3915
+version: 10.18.540.210
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/cumulocity-10-18-540-210-improvement-lwm2m-location-update.md:15]:
```suggestion
version: 10.18.540.209
```

File: content/change-logs/device-management/cumulocity-10-18-540-208-improvement-lwm2m-fw-update.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  - value: component-1KLUzmqfe
+    label: LWM2M
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: DM-3888
+version: 10.18.540.208
+---
+
+In previous LWM2M agent versions, if a device reported an unexpected status or result during a firmware update,
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/cumulocity-10-18-540-208-improvement-lwm2m-fw-update.md:18]: Linking the comment from develop PR
https://github.com/SoftwareAG/c8y-docs/pull/2440/files#r1758637307

Comment (jakubsobolewskisag) [reply content/change-logs/device-management/cumulocity-10-18-540-208-improvement-lwm2m-fw-update.md]: done

File: content/change-logs/device-management/cumulocity-10-18-540-210-improvement-lwm2m-empty-payload-no-ct.md (hunk 2 above)
Comment (jakubsobolewskisag) [reply content/change-logs/device-management/cumulocity-10-18-540-210-improvement-lwm2m-empty-payload-no-ct.md:15]: done

File: content/change-logs/device-management/cumulocity-10-18-540-210-improvement-lwm2m-location-update.md (hunk 3 above)
Comment (jakubsobolewskisag) [reply content/change-logs/device-management/cumulocity-10-18-540-210-improvement-lwm2m-location-update.md:15]: done

**Review by Oezge-Akin — APPROVED** [review]

**Review by aaronraab — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**